Upstream: no, but the issue has been reported
          https://sourceware.org/ml/binutils/2016-01/msg00308.html

From 97b6c31f9e69d6cc947ac0f176a6e92a92524735 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Mon, 1 Feb 2016 21:57:14 +0100
Subject: [PATCH] Set development to false

It has been released and otherwise the date instead of the version is
included in the soname.
---
 bfd/development.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bfd/development.sh b/bfd/development.sh
index 6bbef6d..0e65c07 100644
--- a/bfd/development.sh
+++ b/bfd/development.sh
@@ -16,4 +16,4 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # Controls whether to enable development-mode features by default.
-development=true
+development=false
-- 
2.7.0

